layouts/partials/footer.html

```diff
@@ -11,9 +11,17 @@ <h1 class="site-title">
                         <i class="fa fa-angle-up"></i>
                     </a>
                 </div>
-
-                <p class="footer-copyright">
-                    <span>&copy; {{ .Site.LastChange.Format "2006" }} / Powered by <a href="https://gohugo.io/">Hugo</a></span>
+                
+                <p class="footer-copyright h-card vcard">
+		        	&copy; {{ .Site.LastChange.Format "2006" }} <a style="text-decoration: none" href={{ .Site.BaseURL }} 
+    	         	   class="p-name u-url url author metatag" rel="me">
+        	        	{{ .Site.Author.name }}
+	            	</a> /
+    	            <a class="p-nickname u-email email metatag" rel="me" 
```

Comment:
We need to check if `Site.Author.nick` and `Site.Author.email` are set and only then render this block.

Reply:
Sure, I think this check would be important. I will incorporate the change and resubmit. Thank you!

Reply:
I have added a check for this and made a new commit.